test/integration/verify/controls/azure_container_groups.rb

```diff
@@ -0,0 +1,15 @@
+control 'check container groups' do
+  describe azure_container_groups do
+    it { should exist }
+    its('types') { should include 'Microsoft.ContainerInstance/containerGroups' }
+    its('locations') { should include 'eastus' }
+    its('restart_policies') { should include 'OnFailure' }
+    its('os_types') { should include 'Linux' }
+  end
+end
+
+control 'check all container groups that have been provisioned' do
+  describe azure_container_groups.where(provisioning_state: 'Succeeded') do
```

Comment:
After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

Reply:
It is not mandatory..